### ISSUE TITLE
Specific episode plays for a second and then stops

### ISSUE THREAD

**Issue by kingonly** [amugofjava/anytime_podcast_player#56]
E48 of Podcasting 2.0 plays for a second or two and then stops.

1. Go to the Podcasting 2.0 podcast
2. Play episode 48
3. Episode stops after a second or two

 - Device: OnePlus 8t
 - OS: Android 11

Nope episode is played successfully in other apps.
Dave Jones [confirms](https://twitter.com/PodcastindexOrg/status/1421468626436112384).

**Comment by adamc199** [comment]
Episode plays without problem in Breez on GrapheneOS

**Comment by amugofjava** [comment]
Hi @kingonly,

Yes, I noticed this issue this morning when trying to listen to this episode. There is an issue with the chapters file for this episode. Fetching the chapters throws an exception which results in the playback stopping. I have fixed this locally on the `feature/ios_filesystem` branch and hope to get this complete and merged into master soon. If you wish to fix this sooner on the Breez version the fix is to surround the chapter call in `mobile_podcast_service.dart` with a `try catch` block:

```
  static Future<psearch.Chapters> _loadChaptersByUrlCompute(_FeedComputer c) async {
    psearch.Chapters result;

    try {
      result = await c.api.loadChapters(c.url);
    } catch (e) {
      // Silently ignore the error.
    }

    return result;
  }
```
